src/hooks/http.ts

```diff
@@ -60,13 +66,20 @@ export class Http {
     currentSpan,
     maxSize = getEventEntitySize(true)
   ) {
+    let serviceData: AwsServiceData = {};
     if (body && !requestData.truncated) {
+      serviceData = getAwsServiceData({ ...requestData, body }, null);
```

Comment:
You better do it in the row below.
You need to use `requestData.body + body` and not only `body`.
Please add test on it